### PR TITLE
Add mandyflores xpath scraper

### PR DESCRIPTION
**Pull request by niemands**
Requested in #73 

The details on this site are a nightmare; I have found at least four different styles for them. The commented out selector works for some scenes, but there are scenes, where the description is in the parent span, or in a child span with red color, etc.

### CODE REVIEW

**Comment by bnkai** [comment]
Haven't looked at the whole scraper yet.
Can we gather all the different cases for the description to work something out ?
I spotted these ( selectors are vague and just for illustration purposes for now )

url | selector
------------- | -------------
`https://mandyflores.com/vod/scenes/Zombie-Attack_vids.html` | `//@span[class="update_description"]/div[@align="center"]/span/span` ( red colored details)
`https://mandyflores.com/vod/scenes/Housewife-Eviction-Notice_vids.html` | `//span[class="update_description"]/div[@align="center"]`
`https://mandyflores.com/vod/scenes/Mistletoe-Kiss-Attack_vids.html` | `//@span[class="update_description"]`

for these 3 cases 
```yaml
  sceneScraper:
    common:
      $updateDesc: //span[@class="update_description"]
      $divCenter: /div[@align="center"]
    scene:
      Title: //span[@class="title_bar_hilite"]
      
      Details: $updateDesc$divCenter/span/span/text()|($updateDesc$divCenter/text())[1]|($updateDesc/text())[1]
```
seems to work ok (going from more complex to simpler cases in the union `|` selector makes sure we dont get multiple results )

**Comment by niemands** [comment]
I also found these:
url | selector | extra
----- | ----- | -----
`https://mandyflores.com/vod/scenes/Ill-Tell-Mom_vids.html` | `(//span[@class="update_description"] \| //span[@class="update_description"]/p)/text()[normalize-space(.)]` | concat
`https://mandyflores.com/vod/scenes/Black-Widow-Sisters-Revenge_vids.html` | `//span[@class="update_description"]/div[@align="center"]/span/span[@style]/text()[normalize-space(.)]` | concat

```yaml
  sceneScraper:
    common:
      $updateDesc: //span[@class="update_description"]
      $divCenter: /div[@align="center"]
    scene:
      Details:
        selector: $updateDesc$divCenter/span/span[@style]/text()[normalize-space(.)]|($updateDesc | $updateDesc/p)/text()[normalize-space(.)]|($updateDesc$divCenter/text())[1]|($updateDesc/text())[1]
        concat: "\n"
```
I have expanded your selector, this should work for all previous and new cases

File: scrapers/Mandyflores.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+xPathScrapers:
+  sceneScraper:
+    common:
+      $updateDesc: //span[@class="update_description"]
+      $divCenter: /div[@align="center"]
+    scene:
+      Title: //span[@class="title_bar_hilite"]
+      Details:
+        selector: $updateDesc$divCenter/span/span[@style]/text()[normalize-space(.)]|($updateDesc | $updateDesc/p)/text()[normalize-space(.)]|($updateDesc$divCenter/text())[1]|($updateDesc/text())[1]
+        concat: "\n"
```
Comment (bnkai) [inline scrapers/Mandyflores.yml:16]: Should we switch that to `concat: "\n\n"` ? Imho it maintains better the original structure (sample `https://mandyflores.com/vod/scenes/Ill-Tell-Mom_vids.html`)

File: scrapers/Mandyflores.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      Date:
+        selector: //div[@class="cell update_date"][not(ancestor::span[@class="update_description"])]/text()
```
Comment (bnkai) [inline scrapers/Mandyflores.yml:18]: in some cases the selector returns more than one dates even with the ancestor limitation ( `https://mandyflores.com/vod/scenes/Mistletoe-Kiss-Attack_vids.html` ). The first is always ok though so can keep only that ?
`//div[@class="cell update_date"][not(ancestor::span[@class="update_description"])]/text()[1]`

Comment (niemands) [reply scrapers/Mandyflores.yml:18]: Oh man, I love the structure on this site. Good catch

**Pull request merged**